boxtree/__init__.py

```diff
@@ -20,12 +20,16 @@
 THE SOFTWARE.
 """
 
-from boxtree.tree import (TreeOfBoxes, Tree, TreeWithLinkedPointSources,
-    box_flags_enum)
-from boxtree.tree_of_boxes import (make_tree_of_boxes_root, refine_tree_of_boxes,
-    uniformly_refine_tree_of_boxes, coarsen_tree_of_boxes,
-    refine_and_coarsen_tree_of_boxes, make_meshmode_mesh_from_leaves)
+from boxtree.tree import (Tree, TreeOfBoxes, TreeWithLinkedPointSources,
+                          box_flags_enum)
 from boxtree.tree_build import TreeBuilder
+from boxtree.tree_of_boxes import (coarsen_tree_of_boxes,
+                                   make_meshmode_mesh_from_leaves,
+                                   make_tree_of_boxes_root,
+                                   refine_and_coarsen_tree_of_boxes,
+                                   refine_tree_of_boxes,
+                                   uniformly_refine_tree_of_boxes)
```

Comment:
`isort` seems to have some options for how [multiline imports are handled](https://pycqa.github.io/isort/docs/configuration/multi_line_output_modes.html)?

Reply:
Got a preference?

Reply:
Probably "4 - Hanging Grid"? I like my horizontal space :grin: 
```
from third_party import (
    lib1, lib2, lib3, lib4,
    lib5, ...)
```

Reply:
They have this "centralized config" thing, but... I'm thinking that's overkill? What do you think?

Reply:
Where's that described? Do you mean the "profiles"?

Reply:
Yeah, it seems like they're hardcoded, so probably overkill for now.